yt_dlp/extractor/zingmp3.py

```diff
@@ -120,6 +139,11 @@ def get_api_with_signature(self, name_api, param):
         }
         return f'{self._DOMAIN}{name_api}?{urllib.parse.urlencode(data)}'
 
+    def _entries(self, items):
+        for item in items:
+            if item:
+                yield self.url_result(urljoin(self._DOMAIN, item.get('link')))
```

Comment:
```suggestion
            if item and item.get('link'):
                yield self.url_result(urljoin(self._DOMAIN, item['link']))
```
is this what u meant?